### PR TITLE
Add SonataAdminBundle, SonataBlockBundle and SonataUserBundle.

### PR DESCRIPTION
**Pull request by aLkRicha**
| Q             | A
| ------------- | ---
| License       | MIT

Add SonataAdminBundle, SonataBlockBundle and SonataUserBundle.

### CODE REVIEW

File: sonata-project/admin-bundle/3.30/config/packages/sonata_admin.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+sonata_admin:
+    title: 'Sonata Admin'
```
Comment (OskarStark) [inline sonata-project/admin-bundle/3.30/config/packages/sonata_admin.yaml]: Is this really needed? Do we need to add a config file?

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/DependencyInjection/Configuration.php#L82

ping @greg0ire

File: sonata-project/user-bundle/4.0/config/packages/sonata_user.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+sonata_user:
+    security_acl: true
```
Comment (OskarStark) [inline sonata-project/user-bundle/4.0/config/packages/sonata_user.yaml]: should be `false`

File: sonata-project/admin-bundle/3.30/manifest.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Sonata\\CoreBundle\\SonataCoreBundle": ["all"],
+        "Knp\\Bundle\\MenuBundle\\KnpMenuBundle": ["all"],
+        "Sonata\\AdminBundle\\SonataAdminBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
```
Comment (OskarStark) [inline sonata-project/admin-bundle/3.30/manifest.json]: why do we need this line? and why do we need the `.gitignore` file?

File: sonata-project/admin-bundle/3.30/config/packages/sonata_admin.yaml (hunk 1 above)
Comment (greg0ire) [reply sonata-project/admin-bundle/3.30/config/packages/sonata_admin.yaml]: Not needed no, but changing it is highly recommended, so…

**Comment by OskarStark** [comment]
Is it the right repo, or is this the one it should be: https://github.com/symfony/recipes-contrib

??

**Pull request closed without merging**